integration_tests/mockApis/elite2.ts

```diff
@@ -1506,6 +1506,27 @@ const stubRecategorise = (
   return Promise.all([recategorisationsStub(agencyId), latestOnlyStub()])
 }
 
+const latestOnlyForOffenderStub = ({ offenderNumber }: { offenderNumber: string }) =>
+  stubFor({
+    request: {
+      method: 'GET',
+      url: `/elite2/api/offender-assessments/CATEGORY?offenderNo=${offenderNumber}&latestOnly=false&activeOnly=false`,
```

Comment:
is this expected - latestOnly=false - based on the fn name it reads as if that would be true?